nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -175,17 +176,17 @@ def apply(
 
             channel_axis = node.metatype.output_channel_axis
             if current_bias.ndim > 1:
-                channel_axis = range(current_bias.ndim)[channel_axis]
-                axes = [i for i in range(current_bias.ndim) if i != channel_axis]
-                bias_shift = np.expand_dims(bias_shift, axes)
+                new_shape = [1] * current_bias.ndim
+                new_shape[channel_axis] = bias_shift.shape[0]
+                bias_shift = bias_shift.reshape(new_shape)
```

Comment:
Is this equal change? Setting the channel axis for the new shape may not be the same as the shift's shape 0  element. Also, the expand_dims is not equal to reshape.

Reply:
Revert changes and implement expand_dims to tensor.function. 
https://github.com/openvinotoolkit/nncf/pull/2689/commits/d03191d099fb403e3005aeb4a0ecd79ddbb45148#diff-48a31470e6f5691c217d11e1f28cfcf5dfa25ffdd047ca71353e0ec829b9485eR371-R388